acceptance/tests/invalid_ssl_config.rb

```diff
@@ -4,10 +4,19 @@
 
 agent1 = agents[0]
 
+step 'Setup - Add base certs and config file'
+test_ssl_dir = agent1.tmpdir('test-ssl')
```

Comment:
Lines 8-14 are repeated in each test case now - can this be moved to a helper method? Perhaps configure_host_with_tmpdir_certs(host, cert_number)